src/server/src/main/java/io/cassandrareaper/service/SegmentRunner.java

```diff
@@ -96,6 +96,9 @@
   private int commandId;
   private final AtomicBoolean segmentFailed;
   private final UUID leaderElectionId;
+  private final AtomicBoolean gotFailedNotification;
+  private final AtomicBoolean gotSuccessNotification;
+  private final AtomicBoolean gotCompleteNotification;
```

Comment:
can we use the following shorter names for these plesae?
`completeNotified`, `failedNotified`, `successNotified`

Reply:
sure thing